### PR TITLE
Update to Node 14 & Puppeteer 7

### PR DESCRIPTION
**Pull request by exah**
Tested locally

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "puppeteer": "5.2.1"
+    "puppeteer": "7.0.1"
```
Comment (x80486) [inline package.json:3]: Version `7.0.4` is out...might as well update this one 😉

File: Dockerfile [hunk 2]
```diff
@@ -3,8 +3,7 @@
 # Initially based upon:
 # https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
 
-FROM node:12.18.3-buster-slim@sha256:dd6aa3ed10af4374b88f8a6624aeee7522772bb08e8dd5e917ff729d1d3c3a4f
-    
+FROM node:14.15.4-buster-slim@sha256:8e4843f26333ed01f8733c7163bf88fc31c8516f96c46094d15e906160451f26
```
Comment (x80486) [inline Dockerfile:6]: Just curious: why is the commit hash needed? — I mean, provided the tag won't get updated and so on.

**Comment by x80486** [comment]
Hello there, @jayco! Do you think this one can be merged whenever you have a chance?

File: Dockerfile (hunk 2 above)
Comment (exah) [reply Dockerfile:6]: Not sure. I just followed the same approach as in original file. :)

**Review by rodoabad — APPROVED** [review]

**Review by vipinsaini — APPROVED** [review]

**Comment by jayco** [comment]
Hey, @exah  thanks so much for the PR! 🙏

Thanks, @rodoabad @vipinsaini and @x80486 for looking at this also 🎉

**Pull request merged**